cmd/cartesi-rollups-cli/root/read/epochs/epochs.go

```diff
@@ -0,0 +1,67 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package epochs
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/cartesi/rollups-node/internal/config"
+	"github.com/cartesi/rollups-node/internal/repository"
+	"github.com/cartesi/rollups-node/internal/repository/factory"
+
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "epochs",
+	Short:   "Reads epochs",
+	Example: examples,
+	Run:     run,
+}
+
+const examples = `# Read all reports:
+cartesi-rollups-cli read epochs -n echo-dapp`
+
+var (
+	epochIndex uint64
+)
+
+func init() {
+	Cmd.Flags().Uint64Var(&epochIndex, "epoch-index", 0, "index of the epoch")
+}
+
+func run(cmd *cobra.Command, args []string) {
+	ctx := cmd.Context()
+
+	dsn, err := config.GetDatabaseConnection()
+	cobra.CheckErr(err)
+
+	repo, err := factory.NewRepositoryFromConnectionString(ctx, dsn.String())
+	cobra.CheckErr(err)
+	defer repo.Close()
+
+	var nameOrAddress string
+	pFlags := cmd.Flags()
+	if pFlags.Changed("name") {
+		nameOrAddress = pFlags.Lookup("name").Value.String()
+	} else if pFlags.Changed("address") {
+		nameOrAddress = pFlags.Lookup("address").Value.String()
+	}
+
+	var result []byte
+	if cmd.Flags().Changed("epoch-index") {
+		reports, err := repo.GetEpoch(ctx, nameOrAddress, epochIndex)
```

Comment:
here and in the else branch too.

```suggestion
		epochs, err := repo.GetEpoch(ctx, nameOrAddress, epochIndex)
```